### DISCUSSION TITLE
IOTA firefly balance 0

### DISCUSSION THREAD

**Discussion by NiraksD** [iotaledger/firefly#7580] [Help & Support]
Hello,

Downloaded the latest version of firefly and the balance is 0...even after doing the 'find balances'.
Any suggestions? Last version of firefly was completely fine, a few thousand IOTA on nano ledger s.

Cheers!

**Comment by begonaalvarezd** [comment]
Please keep running the balance finder, its a bit of a patience game, you funds will appear. If you know the wallet you had the funds on you can do a faster balance search from the vesting tab, the "..." button > search rewards or something like that, that will optimize the search for the selected wallet

**Reply by NiraksD** [reply]
Thanks for the reply mate. Got it working in the end! Just needed to restore the wallet from the nano s seed mnemonic (manually, not by clicking in the wallet restore option). Oh joy.

**Reply by begonaalvarezd** [reply]
beautiful, have a good day!
